src/editor/plugins.js

```diff
@@ -104,6 +107,23 @@ export function getPlaceholderPlugin({ text }) {
     }
   }
 
+  function createPlaceholderNode(text) {
+    let span = document.createElement('span');
+    span.innerHTML = `
+<style>
```

Comment:
Ah right - didn't realise that. Could really be fucking with the parser then. 

The original idea was that I could then make the `span` `contenteditable='false'`. The injected content is how it was before this PR, I tried a bunch of different variations - using `pointer-events: none;`, using JS to redo the selection to try force a render of the cursor...this was the best. There was some success with the block element, but the inline was causing problems.